app/models/course.rb

```diff
@@ -72,7 +72,10 @@ class Course < ApplicationRecord
   has_many :staff, -> { where('courses_users.role = 4') },
            through: :courses_users, source: :user
   has_many :survey_notifications, dependent: :destroy
-  has_many :requested_accounts
+  has_many :requested_accounts,
+           class_name: 'RequestedAccount',
+           foreign_key: 'course_id',
```

Comment:
I don't think the `foreign_key` and `class_name` specifications are necessary, as these classes follow the default Rails naming conventions. It's only necessary for cases like `tickets` below, where the class name and/or foreign key column name are different from the defaults that Rails infers.

Reply:
Oops! Sorry for that :)